src/lib/support/Pool.cpp

```diff
@@ -127,6 +127,46 @@ Loop StaticAllocatorBitmap::ForEachActiveObjectInner(void * context, Lambda lamb
     return Loop::Finish;
 }
 
+size_t StaticAllocatorBitmap::FirstActiveIndex()
```

Comment:
Looks like it would have been useful to build as a unit-tested BitSet class.

Reply:
Yes. At the same time though I did add explicit unit tests for the iterator bits that I added, so I believe this is covered (although maybe not as well as a bit-set class would be as this is more than just a bitset)